src/convenience/frameworks.ts

```diff
@@ -390,14 +386,14 @@ const hono: HonoAdapter = (c) => {
         update: c.req.json(),
         header: c.req.header(SECRET_HEADER),
         end: () => {
-            resolveResponse(c.body(null));
+            resolveResponse(c.body(""));
```

Comment:
What does this change?

Reply:
I just copy hono part from main branch to fix broken test